save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/kubernetes/KubernetesManager.kt

```diff
@@ -67,20 +76,48 @@ class KubernetesManager(
                         }
                         // FixMe: Orchestrator doesn't push images to a remote registry, so agents have to be run on the same host.
                         nodeName = System.getenv("NODE_NAME")
+                        initContainers = listOf(
```

Comment:
May be we can extract configuration of this fields for this job into some separated methods, what do you think?
like `initContainers = configureContainers()` and so on?

Reply:
Extracted this one

Reply:
But why only this one?

Reply:
Extracted agent container too